bots/task/test-checklist

```diff
@@ -18,13 +18,15 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+import imp
```

Comment:
I don't think this is being used any more. Doesn't test-static-code complain about the unused import?

Reply:
Done